### PR TITLE
WIP: Clustering Customer Segmantation 

### PR DESCRIPTION
**Pull request by kunjee17**

### CODE REVIEW

**Comment by kunjee17** [comment]
Please don't merge it is still work in progress.

**Comment by kunjee17** [comment]
@mariuszwojcik sir ji, Can you please cross check the code. I am not sure what I am doing is right or not? From language and ML point of view. 

And also there is one TODO there where code is not working? I tried to figure it out but was unable to. It is optional only so as of now I commented one line and skipped it. It would be great if you can have a look at it. 

Let me know if any changes are there or if you like to do the honors please feel free to change the code. 

Thanks :)

**Comment by kunjee17** [comment]
Thanks @mariuszwojcik 👍

**Comment by CESARDELATORRE** [comment]
@kunjee17 - Is this WIP PR still relevant or shall I close it?
Thanks,

**Comment by kunjee17** [comment]
@CESARDELATORRE this is working. But I think code is more look like C# then F# so I told @mariuszwojcik to have a look. If he clears I am good to go.

**Comment by mariuszwojcik** [comment]
@kunjee17 sorry, I was out of the loop for last 2 months. I will have a look into your changes soon. I assume it is still on ML.NET v0.7, correct?

**Comment by CESARDELATORRE** [comment]
@kunjee17 - Not really, I should be thanking you all. In fact, thanks a lot @kunjee17 and @mariuszwojcik , we really appreciate your effort as community collaboration here. 👍

**Comment by CESARDELATORRE** [comment]
Closing this PR since it is stale.

File: samples/fsharp/getting-started/Clustering_CustomerSegmentation/CustomerSegmentation.Train/DataStructures.fs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                        C23 = lookup.["23"].Sum() |> float32
+                        C24 = lookup.["24"].Sum() |> float32
+                        C25 = lookup.["25"].Sum() |> float32
+                        C26 = lookup.["26"].Sum() |> float32
+                        C27 = lookup.["27"].Sum() |> float32
+                        C28 = lookup.["28"].Sum() |> float32
+                        C29 = lookup.["29"].Sum() |> float32
+                        C30 = lookup.["30"].Sum() |> float32
+                        C31 = lookup.["31"].Sum() |> float32
+                        C32 = lookup.["23"].Sum() |> float32
```
Comment (JD-Robbs) [inline samples/fsharp/getting-started/Clustering_CustomerSegmentation/CustomerSegmentation.Train/DataStructures.fs]: 32

**Pull request closed without merging**